webpack.config.js

```diff
@@ -82,6 +82,7 @@ module.exports = (env = {}) => {
     },
     plugins: [
       new webpack.DefinePlugin({
+        __NG_APP_INJECTIONS__: process.env.NG_APP_INJECTIONS ? `'${process.env.NG_APP_INJECTIONS}'` : 'null',
```

Comment:
why process.env? couldn't it be replaced with env.NG_APP_INJECTIONS ?

Reply:
It was inspired by what we've already done in the Public Cloud application.

We could refactoring this by what you are suggested here of course.

Thanks!